_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,163 @@
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/dashboard-index.png" alt="Single-view dashboard in OpenSearch Dashboards" width="600" height="300">
+
+Before you begin this tutorial, make sure you've installed OpenSearch and OpenSearch Dashboards and that you've connected to Dashboards at [http://localhost:5601](http://localhost:5601). The username and password are `admin`.
+{: .note}
+
+## Getting familiar with user interface
+
+In this tutorial you'll learn the basics of creating a dashboard using the **Dashboard** application and OpenSearch sample data. Before getting started, get familiar with the Dashboard user interface. The user interface comprises five main parts, as shown in the following image:
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/user-interface.png" alt="Dashboard user interface" width="600" height="300">
+
+- The **navigation panel** (A) on the left contains the OpenSearch Dashboards applications.
+- The **toolbar** (B) contains frequently used commands and shortcuts.
+- The **search** bar (C) allows you to search for documents and other objects and add filters.
+- The **time filter** (D) allows you to customize the time and date.
+- The **panel** (E) allows you to add existing visualizations to the dashboard or create new ones for the dashboard.
+
+
+_Panel_ is a term used to refer to a visualization displayed on a dashboard. The terms _panel_ and _visualization_ may be used interchangeably throughout this and other Dashboard-related documentation.
+{: .note}
+
+## Adding sample data
+
+If you haven't already added the OpenSearch Dashboards sample datasets as described in [Quickstart guide for OpenSearch Dashboards](https://opensearch.org/docs/latest/dashboards/quickstart-dashboards/), add them before proceeding to the next steps. **Sample eCommerce orders** is used for this tutorial. 
+
+To add the sample dataset:
+
+1. On the OpenSearch Dashboards **Home** page, choose **Add sample data**.
+2. Choose **Add data** and choose **Sample eCommerce orders**.
+
+## Creating dashboards
+
+Now that you've added the sample data, you can create a dashboard using that data. The sample dataset has existing sample visualizations, and you can use those visualizations or create new visualizations for the dashboard. For this tutorial, you'll do both.
+
+1. From the navigation panel, choose **Dashboard**.
+2. From the **Dashboards** window, choose **Create Dashboard**.
+3. Set the time filter to **Last 30 days**.
+4. Choose **Add an existing**.
+5. From the **Add panels** window, choose **Create new** and **Visualization**. This is a shortcut for creating a dashboard and a visualization in one flow. Alternatively, you can use the **Visualize** application to create visualizations and then add them to the dashboard.
+6. From the **New Visualization** window, choose **[eCommerce] Average Sold Quantity**. The donut chart is added to the dashboard.
```

Comment:
These steps don't seem to agree. In this flow we're just adding an existing visualization, so we should choose (click) `[eCommerce] Average Sold Quantity` from the list (or search for it and click). Are we just wanting them to notice that the `Create new` in this flyout is an alternative to the `Create new` button they'll actually use in the next section?

Reply:
Thanks for this input. I was mixing the two tasks - adding an existing viz and creating a new viz. Revised this and the following section.

Reply:
Revised.